### ISSUE TITLE
Recurring events not imported from outlook

### ISSUE THREAD

**Issue by lxtrasher** [sawhney17/logseq-calendars-plugin#26]
Thanks a lot for this great plugin! It's really helpful. However, I'm making heavy use of recurring events in my calendar. Those are not being imported (at least from my outlook, dunno about others). Is this a known issue and planned to be fixed? Do you need me to provide some data (if yes, which)?

Bests,
Alex

**Comment by sawhney17** [comment]
Can you confirm this issue on the version 2.2.1?

If so, it'd  be lovely if you could provide a sample iCal link from outlook.

**Comment by lxtrasher** [comment]
Yes, v2.2.1. Unfortunately I'm not allowed to share the link. Is there anything else I can provide?

**Comment by sunt05** [comment]
> Thanks a lot for this great plugin! It's really helpful. However, I'm making heavy use of recurring events in my calendar. Those are not being imported (at least from my outlook, dunno about others). Is this a known issue and planned to be fixed? Do you need me to provide some data (if yes, which)?

same here 😮‍💨

**Comment by sawhney17** [comment]
I'll  create an outlook account and try replicate this.

**Comment by GeekTeaa** [comment]
Any news on this one? I too have an outlook account with a plethora of re-occurring events that are not getting fetched.

**Comment by GeekTeaa** [comment]
Any updates on this issue?

**Comment by sawhney17** [comment]
Could you tell me your time zone?

**Comment by sawhney17** [comment]
Could you test with a recurring event set for 5 pm and one for 5am. Does any of those show up?

Also does going to yesterdays journal page and clicking result in todays events going to yesterdays page. Make sure to access the journal page using either search or the journals calendar plugin.

**Comment by lxtrasher** [comment]
None of them show up, also not when going to yesterdays page and retrieving those again
I've attached two .ics files (renamed to .txt to be able to upload). "test 5pm nonrep.ics" shows up (non-recurring), but "test 5pm.ics" does not show up (weekly recurring). Maybe this helps

[test 5pm nonrep.ics.txt](https://github.com/sawhney17/logseq-calendars-plugin/files/9140008/test.5pm.nonrep.ics.txt)
[test 5pm.ics.txt](https://github.com/sawhney17/logseq-calendars-plugin/files/9140009/test.5pm.ics.txt)
.

**Comment by Banesmagic** [comment]
Anything on this issue?  It seems that any repeating event that I am invited to in Outlook are not being retrieved.  Recurring events that I create are being retrieved.
